brainreg_napari/sample_data.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+
+import numpy as np
+import pooch
+from napari.types import LayerData
+from skimage.io import imread
+
+base_url = (
```

Comment:
I wonder if it's best to not rely on the URL of the brainreg test data staying stable? We store other BrainGlobe data at [GIN](https://gin.g-node.org/BrainGlobe). In fact, this dataset is already uploaded (albeit to a different organisation): https://gin.g-node.org/cellfinder/data/src/master/brainreg/test_brain.zip

Reply:
This is an excellent idea, and it's probably a good idea more generally to store any data files (test or sample data) on services designed for data storage and sharing instead of github.

Reply:
The BrainGlobe GIN org is likely a better place. I'll upload the data there and post a link.